src/obb/server/interactors/create_lobby_game.clj

```diff
@@ -0,0 +1,19 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.interactors.create-lobby-game
```

Comment:
Having this said, in that case I just didn't come up with something different and am using repositories.